src/ui/Facet/Facet.ts

```diff
@@ -776,6 +776,19 @@ export class Facet extends Component {
     });
   }
 
+  public set currentFacetSearchElementId(id: string) {
+    if (!this.searchContainer) {
```

Comment:
Rather than checking if the `searchContainer` and `id` are defined, and casting the id to a boolean on line 789, I think it would be simpler and more readable to break this function into two: `addFacetSearchAccessibilityAttributes` and `removeFacetSearchAccessibilityAttributes`.

Reply:
Good idea, I'll split this function.